text/0105-dumps-api.md

```diff
@@ -131,18 +99,20 @@ The following table describes which version of the dump correspond to which vers
 | `v4`         | `v0.25.0` and later                  | -                                  |
 
 What does "Highest compatibility dump version" means?
+
 For maintainance reasons, we cannot guarantee the compatibility from old dump versions to the newest ones.
 Concretely, if the user wants to upgrade from Meilisearch `v0.19.0` (dump `v1`) to `v0.26.0` (dump `v4`), migration should be done in two steps. First, import your `v0.19.0` dump into an instance running any version of Meilisearch between v0.21 and v0.24. Second, export another dump from this instance and import it to a final instance running with `v0.26.0`.
 
-## 2. Technical Aspects
+## 4. Technical Aspects
 
-### 2.1. Dump Creation
+### 4.1. Dump Creation
 
-When a dump is being created, the task queue can receive other future operations to perform later but can't process any additional tasks during the dump creation.
+When a dump is being created, the task queue can receive other future operations to perform.
 
-### 2.2. Importing a dump
+### 4.2. Importing a dump
 
 When a dump is being imported, the http API is not available. Meilisearch can't receive read or write requests.
 
-## 3. Future Possibilities
-- Make dump creation a `task`
+## 5. Future Possibilities
+
```

Comment:
As a future possibility, I was thinking; that maybe, at some point, we could create a “fake” task called `DumpImport` that would only be created when you import a dump.
That would allow people to see things like; which updates have been processed since they imported their dump or what was already indexed.

Reply:
I've added it as a future possibility! Thanks @irevoire